jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/TraceEnrichingFilter.java

```diff
@@ -59,6 +61,7 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 Tracer.startSpan(operation, spanId); // caller's span is this span's parent.
             }
         }
+        Tracer.addEvent(Events.serverReceive(System.currentTimeMillis(), TimeUnit.MILLISECONDS));
```

Comment:
I'm tempted to make a span start which records an enum vs have the general event with annotation thing. 

Then, when that span closes we can fire it as an server call span and let the Zipkin converter generate the event details from the span. 

At a minimum, this shouldn't fetch the time again.